schedule/__init__.py

```diff
@@ -157,7 +148,7 @@ def cancel_job(self, job):
         except ValueError:
             logger.debug('Cancelling not-scheduled job "%s"', str(job))
 
-    def every(self, interval=1):
+    def every(self, interval: int = 1) -> "Job":
```

Comment:
This will only work with `from __future__ import annotations` as the class Job is not defined at this point. In Python < 3.8 the annotation is actually evaluated when the signature is parsed. Meaning the class "Job" would have to exist. Or we import the future annotations ... but that will also only do the trick for Python 3.7. There is no way around the quotes for Python 3.6